### DISCUSSION TITLE
2FA does NOT support SMS to Hong Kong (+852)

### DISCUSSION THREAD

**Discussion by changmw** [community/community#102479] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

2FA does NOT support SMS to Hong Kong (+852)!! Could Github add Hong Kong to its SMS service?

The idea of using authenticator app isn't quite the same as using SMS, possibly not as secured. In general, 2FA app requires you to save a generated key (using a QR code) into a password manager. SMS does NOT need a password manager!!

Also, SMS code is arguably more random than the calculated code from 2FA app.

Would like SMS be sent to Hong Kong. Thank you for your attention!

**Comment by changmw** [comment]
It's really strange that China, Hong Kong and Macao are NOT in "Supported countries for SMS authentication", but Taiwan!!!

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication


Related:

SMS service to Hong Kong · community · Discussion #44634
https://github.com/orgs/community/discussions/44634

**Comment by QAQDE** [comment]
Macau is also listed as not supported region as well.
Well done
